src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: (($day-size + $gap-columns) * $columns - $gap-columns);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+    animation: translateY(-20px);
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: '#{$number}';
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $bg-hover;
+      transition: transform 500ms;
```

Comment:
The 'transition' property should be placed with the rest of the static styles, not within the ':hover' pseudo-class. This way, the transition effect will be applied both when the element is hovered over and when the hover is removed.